jujugui/static/gui/src/app/utils/statsd.js

```diff
@@ -29,10 +29,29 @@ const StatsClient = class StatsClient {
     @param {Integer} count The increment amount.
   */
   increase(name, count=1) {
+    name = this._addFlags(name);
     this._send(`${name}:${count}|c`);
   }
 
   /**
+   Adds the active ab test flags to the stat name.
+
+   @param {Array} flags The flags to check for test tags. Used for testing.
+   @param {String} name The stats name.
+  */
+  _addFlags(name, flags=[]) {
+    if (flags.length === 0 && window.juju_config && window.juju_config.flags) {
```

Comment:
Per discussion in IRC we're now passing flags into the constructor so there no longer is a conditional.